app/views/tasks.html

```diff
@@ -102,30 +102,30 @@ <h4>{{'label.heading.pendingtasks' | translate}}</h4>
                             </div>
                             <script type="text/ng-template" id="deletechecker.html">
                                 <div class="modal-header silver">
-                                    <h3 class="bolder">{{'label.heading.delete' | translate}}</h3>
+                                    <h3 class="bolder">{{'label.heading.delete' | translate}}?</h3>
```

Comment:
Use locale-en.json for custom labels instead of directly writing '?' here.

Reply:
@ShruthiRajaram changing label.heading.delete or approve from locale-en.json will change the delete and approve in whole project even in delete button. Otherwise am gonna add new item in lacale-en.json with .deleteoptions and assign it to "Delete'?"